src/f0cal/bootstrap/saltbox/srv/salt/f0cal/bootstrap/user/requirements.txt

```diff
@@ -1,4 +1,4 @@
+{% set skips = salt['pillar.get']("cli:skips") %}
```

Comment:
@AndreyShprengel Didn't we decide it was `cli:skip`?

Reply:
Good catch